crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -378,7 +404,8 @@ def services
           return
         end
 
-        # FIXME: report an error if there is running instances and mode is disruptive
+        status = ::Crowbar::UpgradeStatus.new
+        check_for_running_instances if status.upgrade_mode == "disruptive"
```

Comment:
it's ```status.upgrade_mode == :normal``` (you also don't really need the ```status``` here as Api::Upgrade provides an ```upgrade_mode``` method. But I am repeating myself ;)